### ISSUE TITLE
'Loader' object is not iterable

### ISSUE THREAD

**Issue by basictheprogram** [lamby/django-slack#96]
Installed and configured django-slack and I get the follow exception when I add the `django.template.loaders.app_directories.Loader`, to my TEMPLATES

```
TypeError
'Loader' object is not iterable
/Users/tanner/.local/share/virtualenvs/cribbing-check-N6-Q5UUH/lib/python3.8/site-packages/django/template/context.py, line 244, in bind_template
/Users/tanner/.local/share/virtualenvs/cribbing-check-N6-Q5UUH/bin/python
3.8.5
```
```python
INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'django.contrib.sites',

    'allauth',
    'allauth.account',
    'allauth.socialaccount',
    'allauth.socialaccount.providers.github',

    'django_q',
    'django_slack',
    'storages',
]
```
```python
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates')],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
                'django.template.context_processors.request',

                'django.template.loaders.app_directories.Loader',
            ],
        },
    },
]
```
$ pip3 freeze | grep slack
django-slack==5.15.2

$ pip3 freeze | grep Django
Django==3.1.2

$ python -V
Python 3.8.5

$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.14.6

**Comment by basictheprogram** [comment]
My guess is `django.template.loaders.app_directories.Loader` is not a context_process but a loader and TEMPLATES should look something like this?
```python
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates')],
#        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
                'django.template.context_processors.request',
            ],
            'loaders': [
                'django.template.loaders.filesystem.Loader',
                'django.template.loaders.app_directories.Loader',
            ],
        },
    },
]
```
Any maybe isn't needed in django 3.1?

**Comment by lamby** [comment]
This was poorly worded in the documentation, so I've fixed that by simply removing the Django 1.7 (!) support. Thanks.

**Comment by redskullhydra** [comment]
just make sure that, app_dirs is not set.
